test/storyshots/__snapshots__/storyshots.spec.js.snap

```diff
@@ -2517,6 +2517,136 @@ exports[`Storyshots Paper/Elevation elevation24 1`] = `
 </div>
 `;
 
+exports[`Storyshots PasswordField default 1`] = `
+<div
+  className="MuiFormControl-root"
+>
+  <div
+    className="MuiInput-root MuiInput-formControl MuiInput-underline"
+  >
+    <input
+      aria-invalid={false}
+      className="MuiInput-input MuiInput-inputType"
+      disabled={false}
+      onBlur={[Function]}
+      onChange={[Function]}
+      onFocus={[Function]}
+      required={false}
+      type="password"
+    />
+    <div
+      className="MuiInputAdornment-root MuiInputAdornment-positionEnd"
```

Comment:
Are these lines generated?

Reply:
Yup, I think it's for the `<InputAdornment />`

Reply:
I see. I'll move the `App.js` and and `Root.js` file to another directory instead of creating a subdirectory.